test/runtests.jl

```diff
@@ -17,6 +18,12 @@ using StableRNGs
 
 const testdir = dirname(@__FILE__)
 
+if VERSION >= v"1.9"
+    @testset "Code quality (JET.jl)" begin
+        JET.test_package(Graphs; target_modules=(Graphs,))
```

Comment:
This does not work as expected, and you want to to do `test_package("Graphs"; target_defined_modules=true)` instead. It's obviously better to give a error message when `target_modules` is specified in `[test|report]_package` though.

Reply:
Yeah, it just turned out yesterday that `report_package(::Module, target_modules=(::Module,))` does not work as we expect. This PR still needs the fix on JET.jl (https://github.com/aviatesk/JET.jl/pull/540), so please wait for v0.8.1.

Reply:
Oh right, I thought `target_defined_modules` might use a different mechanism which is unaffected by the issue, but it turns out it just falls back on `target_modules`